libs/transactions/src/lib/views/transaction-details-view/transaction-details-view.component.html

```diff
@@ -1,4 +1,8 @@
-<div class="row" *ngIf="view$ | async as view">
+<div
+  class="row"
+  *ngIf="view$ | async as view"
+  (bbTrackNavigation)="trackNavigation($event)"
+>
```

Comment:
Better not to use `bbTrackNavigation` and `ngIf` together. Wrap everything with `ng-container` and apply `bbTrackNavigation` to the container element.

```
<ng-container (bbTrackNavigation)="trackNavigation($event)">
  <div  class="row" *ngIf="view$ | async as view">
```

Reply:
Done